commit bde84a3b8d02112607cfa33f72b51e39b38790a8
Author: falkTX <falktx@falktx.com>
Date:   Thu Feb 29 00:03:08 2024 +0100

    Fix compat with newer wine that does not have winelib
    
    Signed-off-by: falkTX <falktx@falktx.com>

diff --git a/Makefile.mk b/Makefile.mk
index 991fd5d..67ffafa 100644
--- a/Makefile.mk
+++ b/Makefile.mk
@@ -75,6 +75,7 @@ wineasio_dll_LDFLAGS  = -shared \
 			-L/opt/wine-staging/lib/wine \
 			-L/opt/wine-staging/lib$(M) \
 			-L/opt/wine-staging/lib$(M)/wine
+wineasio_dll_LDFLAGS  += -nostartfiles
 wineasio_dll_DLLS     = odbc32 \
 			ole32 \
 			winmm
